draft-ietf-oauth-jwk-thumbprint-uri.xml

```diff
@@ -129,6 +129,32 @@
         The security considerations of <xref target="RFC7638"/>
 	also apply when using this specification.
       </t>
+
+      <section anchor="MultiplePublicKeysPerPrivateKey" title="Multiple Public Keys per Private Key">
+	<t>
+	  There are cryptographic algorithms for which multiple public keys correspond to the same private key.
+	  This is described in the security considerations of <xref target="RFC7748"/> as follows:
+	</t>
+	<t>
+	  <list style="empty">
+	    <t>
+	      Designers using these curves should be aware that for each public
+	      key, there are several publicly computable public keys that are
+	      equivalent to it, i.e., they produce the same shared secrets.  Thus
+	      using a public key as an identifier and knowledge of a shared secret
+	      as proof of ownership (without including the public keys in the key
+	      derivation) might lead to subtle vulnerabilities.
+	    </t>
+	  </list>
+	</t>
+	<t>
+	  This applies to JWK Thumbprint URIs as identifiers equally as to public keys as identifiers.
```

Comment:
Could we paraphrase this sentence to make it a little bit clearer? Suggest:
> This consideration for public keys as identifiers equally applies to JWK Thumbprint URIs used as identifiers.

Reply:
Agreed